samples/Azure.ResourceManager.Sample/Generated/LongRunningOperation/AvailabilitySetOperationSource.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.Sample
+{
+    internal class AvailabilitySetOperationSource : IOperationSource<AvailabilitySet>
+    {
+        private readonly ArmClient _client;
+
+        internal AvailabilitySetOperationSource(ArmClient client)
```

Comment:
I found this class is not used in this package and I suppose it is because that every operation related with this is a fake LRO.
Is it possible that we do not generate the `OperationSource` classes if the corresponding operations are fake LROs?
But considering these classes are internal, it is OK if we generate those.